boards.txt

```diff
@@ -64,6 +65,49 @@ nRF52DK.menu.lfclk.lfrc.build.lfclk_flags=-DUSE_LFRC
 nRF52DK.menu.lfclk.lfsynt=Synthesized
 nRF52DK.menu.lfclk.lfsynt.build.lfclk_flags=-DUSE_LFSYNT
 
+
+nRF52_minidev.name=nRF52 minimalistic devboard
+
+nRF52_minidev.upload.tool=sandeepmistry:openocd
+nRF52_minidev.upload.protocol=
+nRF52_minidev.upload.interface=stlink-v2
+nRF52_minidev.upload.target=nrf52
+nRF52_minidev.upload.maximum_size=524288
+nRF52_minidev.upload.setup_command=transport select hla_swd; set WORKAREASIZE 0x4000;
+nRF52_minidev.upload.use_1200bps_touch=false
+nRF52_minidev.upload.wait_for_upload_port=false
+nRF52_minidev.upload.native_usb=false
+
+nRF52_minidev.bootloader.tool=sandeepmistry:openocd
+
+nRF52_minidev.build.mcu=cortex-m4
+nRF52_minidev.build.f_cpu=16000000
+nRF52_minidev.build.board=NRF52-minidev
+nRF52_minidev.build.core=nRF5
+nRF52_minidev.build.variant=nRF52_minidev
+nRF52_minidev.build.variant_system_lib=
+nRF52_minidev.build.extra_flags=-DNRF52
+nRF52_minidev.build.float_flags=-mfloat-abi=hard -mfpu=fpv4-sp-d16
+nRF52_minidev.build.ldscript=nrf52_xxaa.ld
+
+nRF52_minidev.menu.softdevice.none=None
+nRF52_minidev.menu.softdevice.none.softdevice=none
+
+nRF52_minidev.menu.softdevice.s132=S132
+nRF52_minidev.menu.softdevice.s132.softdevice=s132
+nRF52_minidev.menu.softdevice.s132.softdeviceversion=2.0.1
+nRF52_minidev.menu.softdevice.s132.upload.maximum_size=409600
+nRF52_minidev.menu.softdevice.s132.build.extra_flags=-DNRF52 -DS132 -DNRF51_S132
+nRF52_minidev.menu.softdevice.s132.build.ldscript=armgcc_s132_nrf52832_xxaa.ld
+
+nRF52_minidev.menu.lfclk.lfxo=Crystal Oscillator
+nRF52_minidev.menu.lfclk.lfxo.build.lfclk_flags=-DUSE_LFXO
```

Comment:
Does the board have an onboard Crystal Oscillator? If yes, we can remove these options ...